components/browser/state/src/main/java/mozilla/components/browser/state/state/content/DownloadState.kt

```diff
@@ -82,4 +84,19 @@ data class DownloadState(
          */
         COMPLETED(6)
     }
+
+    /**
+     * What started the download
+     */
+    sealed class Source {
```

Comment:
nit: As we are using are not using any state, should we use a  enum?

Reply:
Looked through the project and saw other such usages but I'd totally support standardizing enums in these cases. Will change.